Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -95,7 +95,12 @@
     Telecrystal: 16
   categories:
   - UplinkWeapons
-
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
```

Comment:
Change this to "blacklist" if you want it to me removed from nukeops

Reply:
so true